### PR TITLE
fixed 413

### PR DESCRIPTION
**Pull request by dougkenik**
Fix for #413. 

Basically, what was happening, is that when the mod mesh is added as a child to the normal mesh, they are always adjusted vertically (do not know why), and we need to adjust them back down to their original position. This was working before by using a specific routine called setPosition, but there is a conditional in that routine which is now not getting triggered. I changed the routine we use to ensure the translation occurs.

### CODE REVIEW

File: packaging/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "display_name": "Smart Slice",
     "package_id": "SmartSlicePlugin",
     "package_type": "plugin",
-    "package_version": "20.2.20",
+    "package_version": "20.2.201",
```
Comment (bradymadams) [inline packaging/package.json:12]: 20.2.21

**Comment by rickdalgarno** [comment]
@dougkenik How do I get a build of this branch so that I can review this issue?

**Comment by dougkenik** [comment]
> @dougkenik How do I get a build of this branch so that I can review this issue?

I can build you one

**Review by rickdalgarno — APPROVED** [review]
Tested build SmartSlicePlugin-r20.2-413.39 and the mod mesh translation issue is fixed for the part described in #413.

**Review by bradymadams — APPROVED** [review]

**Review by jake-kirkland-teton — APPROVED** [review]

**Review by jake-kirkland-teton — CHANGES_REQUESTED** [review]
Working on another ticket I found another issue with this fix that is a problem:

If you open a project with a mod mesh and then remove the mod mesh, and cancel the changes, the mod mesh is placed under the stage. I assume this will happen in other circumstances as well with our mod meshes.

**Comment by jake-kirkland-teton** [comment]
Mod_Mesh_bug

**Comment by dougkenik** [comment]
@jake-kirkland-teton - do you have the 3MF?

**Comment by jake-kirkland-teton** [comment]
[UMS5_brick_no_mm_3.zip](https://github.com/tetonsim/is-cura-ui/files/5954094/UMS5_brick_no_mm_3.zip)

**Comment by dougkenik** [comment]
> [UMS5_brick_no_mm_3.zip](https://github.com/tetonsim/is-cura-ui/files/5954094/UMS5_brick_no_mm_3.zip)

Thanks - see my other comment too. If you can reproduce in the test build from yesterday, we should create another ticket, otherwise we can assume it's from this fix

**Comment by jake-kirkland-teton** [comment]
I will check on that!

**Comment by jake-kirkland-teton** [comment]
It does indeed happen still in the release candidate.

**Comment by dougkenik** [comment]
> It does indeed happen still in the release candidate.

Ok. Create a new ticket and include the 3MF with steps to reproduce.

**Review by jake-kirkland-teton — APPROVED** [review]

**Pull request merged**